<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerClass</class>
 <widget class="QMainWindow" name="ImageViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>502</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>180</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ver Imágenes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="lwListaImagenes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvImagen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorViewCenter</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="printAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;Ver</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="normalSizeAct"/>
    <addaction name="separator"/>
    <addaction name="fitToWindowAct"/>
    <addaction name="separator"/>
    <addaction name="backImagenAct"/>
    <addaction name="nextImagenAct"/>
    <addaction name="separator"/>
    <addaction name="verListImagesAct"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Barra de Herramientas</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="exitAct"/>
   <addaction name="separator"/>
   <addaction name="openAct"/>
   <addaction name="printAct"/>
   <addaction name="separator"/>
   <addaction name="zoomInAct"/>
   <addaction name="zoomOutAct"/>
   <addaction name="normalSizeAct"/>
   <addaction name="separator"/>
   <addaction name="fitToWindowAct"/>
   <addaction name="separator"/>
   <addaction name="backImagenAct"/>
   <addaction name="nextImagenAct"/>
   <addaction name="separator"/>
   <addaction name="verListImagesAct"/>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/carpeta_1.png</normaloff>:/img16/carpeta_1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Abrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="printAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/imprimir.png</normaloff>:/img16/imprimir.png</iconset>
   </property>
   <property name="text">
    <string>Im&amp;primir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/cerrar.png</normaloff>:/img16/cerrar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/zoom_in.png</normaloff>:/img16/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/zoom_out.png</normaloff>:/img16/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="normalSizeAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/zoom.png</normaloff>:/img16/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Tamaño &amp;Normal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/fit_to_win.png</normaloff>:/img16/fit_to_win.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="backImagenAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/go-back.png</normaloff>:/img16/go-back.png</iconset>
   </property>
   <property name="text">
    <string>Imagen anterior</string>
   </property>
  </action>
  <action name="nextImagenAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/go-next.png</normaloff>:/img16/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Siguiente imagen</string>
   </property>
  </action>
  <action name="verListImagesAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/grlida.qrc">
     <normaloff>:/img16/categorias.png</normaloff>:/img16/categorias.png</iconset>
   </property>
   <property name="text">
    <string>Ver &amp;lista imagenes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/grlida.qrc"/>
 </resources>
 <connections/>
</ui>
